local/bin/py/build/configurations/pull_config.yaml

```diff
@@ -392,3 +392,29 @@
         - "runtime/**/*.md"
       options:
         dest_path: '/security_monitoring/default_rules/'
+
+  - repo_name: datadog-operator
+    contents:
+
+    - action: pull-and-push-file
```

Comment:
the configuration options table in https://docs-staging.datadoghq.com/cswatt/operator2/agent/kubernetes/operator_configuration/ doesn't resize well and leaves a lot of empty space in the first column. Can set the width of the first column to 25% of the view width, and break the option name into multiple lines?